build/python/backend/requirements.txt

```diff
@@ -4,7 +4,7 @@ arc4==0.0.4
 beautifulsoup4==4.9.3
 boltons==20.2.1
 construct==2.10.67
-cryptography==39.0.1
+cryptography==3.4.7
```

Comment:
Is this version needed or can we upgrade here? This was updated to [version 39.0.1](https://github.com/pyca/cryptography/releases/tag/39.0.1)  last week as part of a vulnerability find.

Reply:
I reverted this version to check if it was the cause of the scan_elf bug I encountered. Over time, the bug just disappeared, but this was not the cause. Reverted in a new commit.